### ISSUE TITLE
validate-property-value: false-positive on `background-color` value format

### ISSUE THREAD

**Issue by softdays** [racodond/sonar-css-plugin#120]
Hi, please have a look a the false-positive below:

![sonarqube_issue_backgroundColor_format](https://user-images.githubusercontent.com/948293/68594221-63410600-0497-11ea-89b4-ad094434ea58.png)

@see https://drafts.csswg.org/css-color/#typedef-hex-color

> 8 digits
    The first 6 digits are interpreted identically to the 6-digit notation. The last pair of digits, interpreted as a hexadecimal number, specifies the alpha channel of the color, where 00 represents a fully transparent color and ff represent a fully opaque color.
    In other words, #0000ffcc represents the same color as rgb(0 0 100% / 80%) (a slightly-transparent blue).

**Comment by racodond** [comment]
Hi @softdays,

I don't want to keep maintaining this plugin. Feel free to ping me if you want to take over.

Regards,

David
